.github/workflows/test-setup.yaml

```diff
@@ -18,3 +18,14 @@ jobs:
           method: setup
       - name: Print the artifacts name
         run: echo ${{ steps.artifacts-setup.outputs.name }}
+  test-prolong:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Artifacts Setup
+        id: artifacts-setup
+        uses: ./
+        with:
+          url: ${{ secrets.ARTIFACTS_URL }}
+          user: ${{ secrets.ARTIFACTS_USER }}
+          password: ${{ secrets.ARTIFACTS_PASSWORD }}
+          method: prolong
```

Comment:
I'm writing each method in a different workflow. Let me know if you think we shouldn't.

So if you could create a new workflow `test-prolong.yaml` with the build steps to test the prolong method that would be great.